packages/styles/scss/components/feature-card/_feature-card.scss

```diff
@@ -22,16 +22,6 @@
       padding-top: aspect-ratio(2, 1);
     }
 
-    @include carbon--breakpoint('md') {
-      margin-right: $spacing-05;
-      margin-left: $spacing-05;
-    }
-
-    @include carbon--breakpoint-down('md') {
-      margin-right: -$spacing-05;
-      margin-left: -$spacing-05;
-    }
```

Comment:
These lines are changing `small` and not `medium` layout.

Reply:
missed it, just fixed @RaphaelAmadeu , thank you.